### PR TITLE
feat: updates avatar design and variations

### PR DESCRIPTION
**Pull request by Lelith**
## What changed?

- Contrast borders are used by default
- Legacy default border is not available anymore
- Gray (default) border is thin (1px) while colored borders are thicker (2px)
- Muted colors are now 50% opacity on white background, including the border. If we use the avatar on non-white background, we don't want to see that background color. For that reason, I use color-mix with 50% white.

## Screenshots

image

### CODE REVIEW

**Comment by damann** [comment]
> Tiny muted doesn't have a border
Why did we decide to remove the border from tiny? FWIW I'm thinking about removing the tiny variant all together (in a follow up).

**Comment by andipaetzold** [comment]
> > Tiny muted doesn't have a border
> 
> Why did we decide to remove the border from tiny? FWIW I'm thinking about removing the tiny variant all together (in a follow up).

@damann I followed the provided designs. But I think that aspect was only temporary. I'll re-add the border.

**Comment by damann** [comment]
> @damann I followed the provided designs. But I think that aspect was only temporary. I'll re-add the border.

@andipaetzold My apologies, I didn't prepare the avatar component diligently enough. I hate causing double work, especially if it's mundane task such as this.

File: packages/components/avatar/src/Avatar/utils.ts [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
+  strawberry: '#FF4C95',
+  orange: tokens.orange500,
+  gold: tokens.yellow700,
+  olive: '#747826',
+  green: tokens.green500,
+  ocean: '#007F9B',
 };
+
+export function applyMuted(color: string): string {
+  return `color-mix(in srgb, ${color}, ${tokens.colorWhite} 50%)`;
```
[the remark this answers is not in the document]
Comment (andipaetzold) [reply packages/components/avatar/src/Avatar/utils.ts:20]: argh....

[the remark this answers is not in the document]
Comment (andipaetzold) [reply packages/components/avatar/src/Avatar/utils.ts]: I implemented `color-mix` in js now until Safari supports it (September 2025), wdyt?

[the remark this answers is not in the document]
Comment (andipaetzold) [reply packages/components/avatar/src/Avatar/utils.ts]: Alternative would be about adding another container with a background etc.

File: packages/components/avatar/stories/Avatar.stories.tsx [hunk 2]
```diff
@@ -25,80 +29,97 @@ export const Overview: Story<AvatarProps> = (args) => {
...
         <Avatar {...args} size="large" variant="app" colorVariant="muted" />
-        <Avatar {...args} size="large" colorVariant="pink" variant="app" />
+        <Avatar {...args} size="large" variant="app" colorVariant="pink" />
         <Avatar {...args} size="large" variant="app" colorVariant="lavender" />
         <Avatar {...args} size="large" variant="app" colorVariant="emerald" />
       </Flex>
 
-      <SectionHeading as="h4">Tooltip properties</SectionHeading>
+      <SectionHeading as="h3" marginBottom="spacingS">
+        Tooltip properties
```
Comment (denkristoffer) [inline packages/components/avatar/stories/Avatar.stories.tsx:121]: @andipaetzold Can we force this tooltip open for the storybook?

**Review by denkristoffer — CHANGES_REQUESTED** [review]
The new Avatar looks bad on the old navigation, should we merge this to `next` and release an alpha?

https://5fd1dda724cc620021ace8c5-oxrqgsaxju.chromatic.com/?path=/story/components-navbar--basic

File: packages/components/avatar/stories/Avatar.stories.tsx (hunk 2 above)
Comment (andipaetzold) [reply packages/components/avatar/stories/Avatar.stories.tsx:121]: Not sure how to technically achieve that.

**Review by denkristoffer — APPROVED** [review]

**Pull request merged**